### ISSUE TITLE
ResNet18-Cifar100 pretrain parameter request

### ISSUE THREAD

**Issue by MushrO-om** [csm9493/L2UL#2]
Thanks for your inspiring and clear code!
I wonder if you used the same training parameter to train the pretrained resnet18 for CIFAR100 dataset, as in ResNet50-CIFAR100?
`optimizer = optim.SGD(model.parameters(), lr=0.01,
                          momentum=0.9, weight_decay=5e-4)
    scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optimizer, T_max=200)`

**Comment by csm9493** [comment]
Yes. We did it.

**Comment by MushrO-om** [comment]
Thanks for your reply!
